src/OrleansServiceBus/Providers/Streams/EventHub/EventHubQueueCache.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Orleans.ServiceBus.Providers
 {
-    public abstract class EventHubQueueCache<TCachedMessage> : IEventHubQueueCache
+    public abstract class EventHubQueueCacheBase<TCachedMessage> : IEventHubQueueCache
```

Comment:
Can we avoid 'Base'? Such a name suffix is frown upon these days. And it's another public class with no docs.